lib/api/controllers/documentController.js

```diff
@@ -729,6 +730,11 @@ class DocumentController extends NativeController {
 
     this.assertNotExceedMaxWrite(documents.length);
 
+    if (Object.prototype.hasOwnProperty.call(request.input.args, '_source') && 
+        !request.getBoolean('_source')) {
+      _source = false;
+    }
```

Comment:
```suggestion
    const _source = request.getBoolean('_source');
```

Unlike other methods `getBoolean` does not throw if the argument is missing and returns false instead, using this is far more simpler and readable.

Reply:
Yes, but I did that because by default `getBoolean` returns `false` if the argument is missing, and I want it to returs `true` if the `_source` param is missing. Is there a function for know if the param exist by default ?

Reply:
You must check if the argument exists in `request.input.args.source`, if it doesn't then default value is true, if it does then the is considered as false only if it's `false` (the boolean` or `"false"` (the string)